### PR TITLE
Update IABot to use DeadlinkChecker library

### PR DESCRIPTION
**Pull request by Niharika29**

### CODE REVIEW

File: IABot/Parser/parse.php [hunk 1]
```diff
@@ -34,6 +34,9 @@
 * @license https://www.gnu.org/licenses/gpl.txt
 * @copyright Copyright (c) 2016, Maximilian Doerr
 */
+require __DIR__ . '/../../vendor/autoload.php';
```
Comment (cyberpower678) [inline IABot/Parser/parse.php]: I am unfamiliar with this file.  Why is it not where the other files are included?

File: IABot/Parser/parse.php [hunk 2]
```diff
@@ -34,6 +34,9 @@
...
+use Wikimedia\DeadlinkChecker\CheckIfDead;
```
Comment (cyberpower678) [inline IABot/Parser/parse.php]: This appears to be called twice.

File: IABot/Parser/parse.php (hunk 1 above)
Comment (Niharika29) [inline IABot/Parser/parse.php]: This is required by composer to load CheckIfDead class. I *suspect* we can do away with this line keeping only the 'use Wikimedia\DeadlinkChecker\CheckIfDead;' statement, but someone needs to test it.

File: IABot/Parser/parse.php (hunk 2 above)
Comment (Niharika29) [inline IABot/Parser/parse.php]: Twice?

File: IABot/deadlink.config.inc.php [hunk 3]
```diff
@@ -79,7 +79,7 @@
 	if( $multithread || $workers ) require_once( 'thread.php' );
 	require_once( 'Parser/parse.php' );
 	require_once( 'DB.php' );
-	require_once( 'checkIfDead.php');
+	use Wikimedia\DeadlinkChecker\CheckIfDead;
```
Comment (cyberpower678) [inline IABot/deadlink.config.inc.php]: This is the first instance of the call.

File: IABot/Parser/parse.php (hunk 1 above)
Comment (Niharika29) [inline IABot/Parser/parse.php]: Wait, what do you mean by "Why is it not where the other files are included?"?

Comment (cyberpower678) [inline IABot/Parser/parse.php]: Why is it not in the config file where all the other files are loaded?

Comment (Niharika29) [inline IABot/Parser/parse.php]: I didn't know all files are loaded in the config. I'll update this.

Comment (cyberpower678) [inline IABot/Parser/parse.php]: No worries.  I was just wondering. :-)

**Comment by kaldari** [comment]
+2, looks good. Tested.

**Pull request merged**